index.js

```diff
@@ -53,17 +53,20 @@ Allure.prototype.endCase = function(status, err, timestamp) {
 Allure.prototype.startStep = function(stepName, timestamp) {
     var step = new Step(stepName, timestamp),
         suite = this.getCurrentSuite();
-    step.parent = suite.currentStep;
-    step.parent.addStep(step);
-    suite.currentStep = step;
+    if (suite.currentStep) {
```

Comment:
there should be `if(suite && suite.currentStep)`. There may be a case when there is no suite too.